.github/workflows/build-subsystems.yaml

```diff
@@ -92,6 +100,7 @@ jobs:
       - build-measurements
       - build-oidc-mock
       - build-transfer
+      - build-openapi
```

Comment:
I don't think you need another workflow/job to build an openapi spec. Could we instead strive to incorporate it into the regular build workflows?

Reply:
Its an easy way to get all files in one docker image